.jenkins/jenkins_buildbot_python2_debug.sh

```diff
@@ -9,6 +9,40 @@ export PATH=/usr/local/cuda/bin:$PATH
 export LD_LIBRARY_PATH=/usr/local/cuda/lib64:$LD_LIBRARY_PATH
 export LIBRARY_PATH=/usr/local/cuda/lib64:$LIBRARY_PATH
 
+GPUARRAY_CONFIG="Release"
```

Comment:
It's ok, we always use Release for buildbots. The alternatives are for development of libgpuarray itself.